.github/workflows/staging-build-tar-temp.yml

```diff
@@ -0,0 +1,81 @@
+name: Process TAR Artifacts
```

Comment:
It looks like staging-build-tar-temp.yml duplicates a lot of the changes to staging-build-tar.yml. Is that intentional?

Reply:
Testing file, will remove once everything is ready.